EIPS/eip-6909.md

```diff
@@ -0,0 +1,600 @@
+---
+eip: 6909
+title: Multi-Token
+description: A minimal specification for managing multiple tokens by their id in a single contract.
+author: Joshua Trujillo (@jtriley)
+discussions-to: https://ethereum-magicians.org/t/eip-6909-multi-token-standard/13891
+status: Draft
+type: Standards Track
+category: ERC
+created: 2023-04-19
+requires: 165
+---
+
+## Abstract
+
+The following standard specifies a multi-token contract as a simplified alternative to the [ERC-1155](./eip-1155.md) Multi-Token Standard.
+
+## Motivation
+
+The ERC-1155 standard includes unnecessary features such as requiring recipient accounts with code to implement callbacks returning specific values and batch-calls in the specification. In addition, the single operator permission scheme grants unlimited allowance on every token ID in the contract. Simplification of the transfer scheme, the addition of a more granular and familiar permission scheme, and minor changes to the token URI specification from the ERC-1155 standard are key features of the following specification. Backwards compatibility is deliberately removed only where necessary. Additional features such as batch calls, increase and decrease allowance methods, and other user experience improvements are deliberately omitted in the specification to minimize the required external interface.
```

Comment:
Might be nice to see some explanation about why you feel the callbacks and batch functions are unnecessary. Like, are there tokens out there that weren't released because of that extra development time?

Reply:
expanded on explanations